app/services/cycle_timetable.rb

```diff
@@ -180,7 +187,7 @@ def self.before_find_reopens?
   end
 
   def self.before_apply_reopens?
-    return true if Time.zone.now.to_date <= apply_reopens.beginning_of_day
+    return true if Time.zone.now.to_date <= CYCLE_DATES[next_year][:apply_opens].beginning_of_day
```

Comment:
```suggestion
    return true if Time.zone.now.to_date <= apply_reopens.beginning_of_day
```

What should this method return if we're already into the new cycle? Do we need to consider `:apply_opens` in this cycle too?

Reply:
`.before_apply_reopens?` is called from the carry over interstitials (and nowhere else currently) - this needs to compare the current time versus when apply reopens and we do not need to consider when apply opened (yet!)

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5106/commits/e34524361f6910fef5dcb5eb999fa32c2d40da5e